### PR TITLE
Disable assert and remove use of hasOwnProperty -- they are slow

### PR DESCRIPTION
**Pull request by dstoc**

### CODE REVIEW

File: web-animations.js [hunk 1]
```diff
@@ -2063,7 +2065,7 @@ KeyframeAnimationEffect.prototype = createObject(AnimationEffect.prototype, {
       }
       this._cachedProperties = [];
       for (var p in properties) {
-        if (properties.hasOwnProperty(p)) {
+        if (p in properties) {
```
Comment (alancutter) [inline web-animations.js]: Do we still need this check?

Comment (dstoc) [inline web-animations.js]: Nope! Refactored.

File: web-animations.js [hunk 2]
```diff
@@ -2096,22 +2096,22 @@ KeyframeInternal.prototype = {
     return this.rawValues[property];
   },
   addPropertyValuePair: function(property, value) {
-    console.assert(!this.cssValues.hasOwnProperty(property));
+    ASSERT_ENABLED && console.assert(!this.cssValues.hasOwnProperty(property));
     this.cssValues[property] = value;
   },
   hasValueForProperty: function(property) {
-    return this.cssValues.hasOwnProperty(property);
+    return property in this.cssValues;
```
Comment (alancutter) [inline web-animations.js:2103]: I would add an assert here that hasOwnProperty() matches 'in' to be safe.

Comment (dstoc) [inline web-animations.js:2103]: It does, since we initialize (line 2086) and populate (line 2100) cssValues ourselves

Comment (alancutter) [inline web-animations.js:2103]: I see, all good then.

**Pull request merged**